whois-api/src/main/java/net/ripe/db/whois/api/httpserver/RemoteAddressFilter.java

```diff
@@ -54,10 +47,8 @@ public String getRemoteAddr() {
             return remoteAddress;
         }
 
-        //TODO: Ipv6 address will have brackets by default. Make changes to ipv6 parser instead and check all usages of getRemoteAddress
         private static String getRemoteAddress(final HttpServletRequest request) {
```

Comment:
minor: since all the login has been removed, perhaps drop the method totally and just have:

```
this.remoteAddress = request.getRemoteAddr();
```

in the constructor